performance_test/src/utilities/rt_enabler.hpp

```diff
@@ -96,11 +96,10 @@ inline int32_t proc_rt_init(const uint32_t cpu_bit_mask_in, const int32_t prio)
 
   if (proc_rt_info.is_rt) {
     //
-    // Lock the current and future memory in RAM.
-    // NOTE: The entire process mem should be allocated during init phase.
-    // Since that is not the case now, locking the future memory
-    //
-    res = mlockall(MCL_CURRENT | MCL_FUTURE);
+    // Lock the current memory in RAM.
+    // Locking future memory is not supported by many middleware implementations 
```

Comment:
This description is not accurate. It is not about support in middleware implementation, it is more about for this particular test application, where based on the settings, more virtual memory  might get dynamically allocated and locking all of it might run out of physical memory.

Reply:
@anup-pem: Could you provide me with a more accurate description I could place there instead?

Reply:
Based on various QoS settings, certain middleware implementaions, allocates excessive virtual memory dynamically. Locking this excessive virtual memory into physical memory will cause running out of physical memory. Therefore, MCL_FUTURE option is disabled.

Reply:
Thank you, I added this.